.mdlrc

```diff
@@ -0,0 +1 @@
+## markdownlint config
```

Comment:
Is this file needed?

Reply:
ok, figured it out, `markdownlint-cli` has a different config than `markdownlint`. This doc was critical to figuring it out. https://lyz-code.github.io/blue-book/devops/markdownlint/